app/helpers/govuk_design_system_helper.rb

```diff
@@ -7,13 +7,14 @@
 require 'gds_design_system_breadcrumb_builder'
 
 module GovukDesignSystemHelper
-  def govuk_page_title(title = nil, caption = nil, tag_options = {})
-    content_for :page_title, page_title(title, caption)
+  def govuk_page_title(heading = nil, caption = nil, options = {})
+    title = options.delete(:title)
```

Comment:
it looks a bit strange that `title`, wich is one important param of this method, is "hidden" inside `options`
is not the caption something optional?